websocket/client.go

```diff
@@ -320,6 +320,14 @@ func (c *Conn) Start() {
 	go c.run()
 }
 
+func (c *Conn) cloneEventHandlers() (handlers []SpeakerEventHandler) {
+	c.RLock()
+	handlers = append(handlers, c.eventHandlers...)
```

Comment:
should be Lock() as you writing, only one instance should have access to this ressource

Reply:
there is no writing to c.eventHandlers here, only read/copy